keyboards/montex/keymaps/default/config.h

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2021 NachoxMacho
```

Comment:
This file should be moved to `keyboards/montex/config.h`.

```
git mv keyboards/montex/keymaps/default/config.h keyboards/montex/config.h
```

Reply:
Moved